amd64-hassio-addon/run.sh

```diff
@@ -10,22 +10,45 @@ HASSIO_DEBUG_OTAU="$(jq --raw-output '.debug_otau' $CONFIG_PATH)"
 HASSIO_DECONZ_WEB_PORT="$(jq --raw-output '.web_port' $CONFIG_PATH)"
 HASSIO_DECONZ_WS_PORT="$(jq --raw-output '.websockets_port' $CONFIG_PATH)"
 HASSIO_DECONZ_DEVICE="$(jq --raw-output '.deconz_device' $CONFIG_PATH)"
+HASSIO_DECONZ_VNC_MODE=":$(jq --raw-output '.vnc.active' $CONFIG_PATH)"
+HASSIO_DECONZ_VNC_DISPLAY=":$(jq --raw-output '.vnc.port - 5900' $CONFIG_PATH)"
```

Comment:
Does this:
```
HASSIO_DECONZ_VNC_DISPLAY=":$(jq --raw-output '.vnc.port - 5900' $CONFIG_PATH)
```
Set 5900 as the default? If so, we don't need to do that here, in the Hass.io settings we can specify the default value.

Reply:
Port 5900 corresponds to display 0, 5901 display 1, ... This defines the offset. A better way would be to always use 5900 internally and map it to an external port, but as we have to use host networking this is the way to go. Another option would be to define the display number instead and calculate the port from the display

Reply:
I guess my only question then is: we are allowing people to set the port #, so what if someone sets the port at 7000 - they would get display 1100 - is that OK?

Reply:
Only ports smaller than 5900 seem to be a problem. In the related hassio-addons pull request I limited it to 5900-5999.

Reply:
OK great.